### DISCUSSION TITLE
Unity plugin support

### DISCUSSION THREAD

**Discussion by Rakiah** [SharpMap/SharpMap#207] [Q&A]
Hello,

Is there any kind of support for Unity3d?

**Comment by FObermaier** [comment]
There are no components specifically for  Unity3D in this project, if that is what you mean.

**Comment by Rakiah** [comment]
Yes that was kind of my question, just wondering if there is any documentation somewhere apart from the sample code ? Otherwise do you know how hard would it be to connect with a Unity renderer ? Basically if SharpMap could provide a way to output raw RGB directly from vector definitions that wouldnt be too hard to implement a unity plugin

**Comment by FObermaier** [comment]
There is a [SharpDX layer](https://github.com/SharpMap/SharpMap/tree/Branches/1.0/SharpMap.Layers.SharpDX), is that kind of what you are looking for?
